scipy/stats/tests/test_morestats.py

```diff
@@ -998,6 +998,14 @@ def test_invalid_ci_method(self):
         with pytest.raises(ValueError, match="method must be"):
             res.proportion_ci(method="plate of shrimp")
 
+    def test_alias(self):
+        res = stats.binomtest(3, n=10, p=0.1)
+        assert_equal(res.proportion_estimate, res.statistic)
```

Comment:
I would change the other tests to test against `res.statistic` instead of `res.proportion_estimate`

Reply:
OK, done.

Reply:
@tupui Do you think we need this to really be an "alias" or is a separate attribute with the same value OK? The latter would simplify the code (and not make me cringe so much to keep the old name around).